app/views/projects/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
I think you understand now after coming out of the MVC in Action class why we wouldn't want to do a `count` in the view.